EateryModel/Sources/EateryModel/Model/EateryAlert.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  EateryAlert.swift
+//  
+//
+//  Created by William Ma on 1/28/22.
+//
+
+import Foundation
+
+public struct EateryAlert: Codable, Hashable {
+
+    public let description: String?
+
+    public let endTimestamp: Int
+
+    public let id: Int64
+
+    public let startTimestamp: Int
+
```

Comment:
I noticed these are alphabetized. Not sure if that was a coincidence

Reply:
I think in both the EateryModel and EateryGetAPI packages I ordered all public fields alphabetically, but I'm not convinced of this. For now though it's just meant to be consistent

Reply:
Got it